dsl/src/main/scala/org/http4s/dsl/impl/ResponseGenerator.scala

```diff
@@ -21,13 +21,18 @@ trait ResponseGenerator extends Any {
   * }}}
   */
 trait EmptyResponseGenerator[F[_]] extends Any with ResponseGenerator {
-  def apply()(implicit F: Applicative[F]): F[Response[F]] = F.pure(Response(status))
+  def apply()(implicit F: Applicative[F]): F[Response[F]] =
+    F.pure(Response(status))
+
+  def apply(headers: Header*)(implicit F: Applicative[F]): F[Response[F]] =
```

Comment:
Now that I look at it, this probably doesn't need to be an overload.

Reply:
It will just be an empty list then, right?

Reply:
Yes.